code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -528,24 +528,24 @@
 	if(istype(passed_mob, /mob/living/carbon/human))
 		var/mob/living/carbon/human/passed_human = passed_mob
 		if (issynth(passed_human))
-			return 1
+			return TRUE
 		switch(hudtype)
 			if("security")
 				if(skillcheck(passed_human, SKILL_POLICE, SKILL_POLICE_SKILLED))
 					var/datum/mob_hud/sec_hud = GLOB.huds[MOB_HUD_SECURITY_ADVANCED]
-					if(locate(passed_mob) in sec_hud.hudusers)
+					if(sec_hud.hudusers[passed_human])
```

Comment:
These fixes aren't exclusive to zombies; changelog should have another line or just more generically explain that HUDs are being fixed - not merely for zombies.

Reply:
ok